tests/roots/test-ext-viewcode-find-package/conf.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
+
+source_dir = os.path.abspath('.')
+if source_dir not in sys.path:
+    sys.path.insert(0, source_dir)
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode']
+exclude_patterns = ['_build']
+
+
+if 'test_linkcode' in tags:  # NOQA
```

Comment:
what error are you suppressing here?

Reply:
That line was in the other test as well: https://github.com/sphinx-doc/sphinx/blob/5.x/tests/roots/test-ext-viewcode/conf.py

I've removed both instances, and there doesn't appear to be any errors as a result. Maybe it was required at some point in the past, but it doesn't appear to be required now.

Reply:
@danieleades That `# NOQA` was for some old error that is no longer present